mesh/core/src/main/protobuf/resolver.proto

```diff
@@ -0,0 +1,85 @@
+syntax = "proto3";
+
+package io.linkerd.mesh;
+
+import "dtab.proto";
+
+/**
+ * Supports resolution of a concrete name to a replica set.
+ *
+ * Status: Experimental; this API is not yet finalized.
+ */
+service Resolver {
+
+  rpc GetReplicas (ReplicasReq) returns (Replicas) {}
```

Comment:
Yeah, I'm trying to use clearer language (and definitely move away from opaque finagle terms like Name, Addr, Address).  _Replicas_ explicitly indicates that the returned set of endpoints can be used interchangeably and that this API does not support other schemes (i.e. by surfacing shard IDs, etc).